proto/protovalidate-testing/buf/validate/conformance/cases/custom_constraints/custom_constraints.proto

```diff
@@ -162,13 +162,48 @@ message FieldExpressionMessage {
     int32 a = 1;
   }
 }
-message FieldExpressionMapScalar {
+message FieldExpressionMapInt32 {
   map<int32, int32> val = 1 [(buf.validate.field).cel = {
     id: "field_expression.map.scalar"
     message: "test message field_expression.map.scalar"
     expression: "this.all(k, this[k] == 1)"
   }];
 }
+message FieldExpressionMapInt64 {
+  map<int64, int64> val = 1 [(buf.validate.field).cel = {
+    id: "field_expression.map.scalar"
+    message: "test message field_expression.map.scalar"
```

Comment:
I left these ids and messages the same for all of them since tests just verify the text. But I can change them if we'd rather them be more specific.

Reply:
totally fine!